<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnsleDictDialog</class>
 <widget class="QDialog" name="AnsleDictDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniAnsleDict</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/new/prefix2/rc/Wordpress.png</normaloff>:/new/prefix2/rc/Wordpress.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4" columnstretch="2,5">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QPushButton" name="ConfigPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>配置发音方式</string>
           </property>
           <property name="statusTip">
            <string>配置发音方式</string>
           </property>
           <property name="whatsThis">
            <string>配置发音方式</string>
           </property>
           <property name="accessibleDescription">
            <string>配置发音方式</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="rc.qrc">
             <normaloff>:/new/prefix2/rc/Gear.ico</normaloff>:/new/prefix2/rc/Gear.ico</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>16</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="AddToNewWordPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>加入生词本</string>
           </property>
           <property name="statusTip">
            <string>加入生词本</string>
           </property>
           <property name="whatsThis">
            <string>加入生词本</string>
           </property>
           <property name="accessibleDescription">
            <string>加入生词本</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="rc.qrc">
             <normaloff>:/new/prefix2/rc/Wait.ico</normaloff>:/new/prefix2/rc/Wait.ico</iconset>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="SpeakTTSPushButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>发音</string>
           </property>
           <property name="statusTip">
            <string>发音</string>
           </property>
           <property name="whatsThis">
            <string>发音</string>
           </property>
           <property name="accessibleDescription">
            <string>发音</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="rc.qrc">
             <normaloff>:/new/prefix2/rc/Sound.ico</normaloff>:/new/prefix2/rc/Sound.ico</iconset>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="WordTableView">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>0</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="toolTip">
          <string>单词列表</string>
         </property>
         <property name="statusTip">
          <string>单词列表</string>
         </property>
         <property name="whatsThis">
          <string>单词列表</string>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="WordLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="rc.qrc">:/new/prefix2/rc/Wordpress.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="WordLineEdit">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>输入需要查询的单词</string>
           </property>
           <property name="statusTip">
            <string>输入需要查询的单词</string>
           </property>
           <property name="whatsThis">
            <string>输入需要查询的单词</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="ExpComboBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip">
            <string>选择匹配方式</string>
           </property>
           <property name="statusTip">
            <string>选择匹配方式</string>
           </property>
           <property name="whatsThis">
            <string>选择匹配方式</string>
           </property>
           <item>
            <property name="text">
             <string>头匹配</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>中匹配</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>尾匹配</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTextBrowser" name="MeanTextBrowser">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="toolTip">
          <string>单词词义</string>
         </property>
         <property name="statusTip">
          <string>单词词义</string>
         </property>
         <property name="whatsThis">
          <string>单词词义</string>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <action name="action_AddToNewWord">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/prefix2/rc/Wait.ico</normaloff>:/new/prefix2/rc/Wait.ico</iconset>
   </property>
   <property name="text">
    <string>添加到生词本</string>
   </property>
   <property name="toolTip">
    <string>添加到生词本</string>
   </property>
  </action>
  <action name="action_Speak">
   <property name="icon">
    <iconset resource="rc.qrc">
     <normaloff>:/new/prefix2/rc/Sound.ico</normaloff>:/new/prefix2/rc/Sound.ico</iconset>
   </property>
   <property name="text">
    <string>发音</string>
   </property>
   <property name="toolTip">
    <string>单词发音</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
